addons/pos_gift_card/static/src/css/giftCard.css

```diff
@@ -34,10 +34,17 @@
 }
 
 .giftCardPopupButton {
-    width: 50% !important;
-    margin: 6px;
+    width: auto !important;
+    height: auto !important;
+    padding: 10px;
 }
 
 .giftCardPopupConfirmButton {
-    width: 40% !important;
+    flex-grow: 1;
+    height: auto !important;
+    padding: 10px;
+    display: flex;
```

Comment:
same here for display, justify, align and line height should be applied to both or none